components/card/profile/src/_settings.scss

```diff
@@ -0,0 +1,11 @@
+$bd-card-profile-avatar-halo: 4px solid $c-accent !default;
+$bgc-card-profile: $c-gray-lightest !default;
+$c-profile-info: $c-gray !default;
+$ff-profile-rating-info-box-name: 'Open Sans', sans-serif !default;
+
+$h-card-profile: 280px;
+$w-card-profile: 172px;
```

Comment:
what do you think about the component adapt to its parent? IMO we should try to avoid fixing withs and heights.